### DISCUSSION TITLE
I want to add my mii but I can't !!!

### DISCUSSION THREAD

**Discussion by RHAP-11** [RiiConnect24/RiiTag-Next#59] [Q&A]
hi there, I have a problem,
I wanted to add my mii as my avatar with his qr code that I created with my 2DS but it doesn't work ! I don't know if is because it's European but the picture is JPG… Thanks for help me !
My qr code : 
![HNI_0021](https://github.com/RiiConnect24/RiiTag-Next/assets/134954749/91306e2e-5008-479f-97a4-29d809a089a6)
![Capture d’écran 2023-12-10 185320](https://github.com/RiiConnect24/RiiTag-Next/assets/134954749/236de91e-b30f-4ac7-afa9-d8c1175f60aa)
![Capture d’écran 2023-12-10 185722](https://github.com/RiiConnect24/RiiTag-Next/assets/134954749/b96e5eee-b871-44ca-9db4-917301180d87)

**Comment by larsenv** [comment]
That's odd, I checked the logs, seems that it didn't specify the type of Mii that it passes in, resulting in it giving that error. I set it to read from 3DS/Wii U by default now, can you try again?

**Reply by RHAP-11** [reply]
I tried but it still doesn't work... it wrotes "This does not appear to be a supported Mii"...

**Reply by iWockstar** [reply]
I have the same issue but I'm located in the United States
